config_test.go

```diff
@@ -226,3 +243,25 @@ func TestLoadFromValueUnmarshaler(t *testing.T) {
 	require.Equal(t, 10, s.Age)
 	require.Zero(t, s.Ignored)
 }
+
+func TestBackendTag(t *testing.T) {
```

Comment:
With this test we are not certain that the `backend` tag only queries the right backend. Also, what about the `required` tag ?

Reply:
Of course yes, if it queries the wrong backend the assert fails. Isn'it ?

Reply:
> what about the required tag ?

What do you mean ?

Reply:
The assert will fail but we don't know if:
- Confita only queried the specified backend
- OR Confita queried all the backends
In both cases the assert will fail so this assertion is not enough. We need a way to be certain that *only one* backend has been called.

Reply:
> what about the required tag

What about the `required` option ?

Reply:
You mean we need to add the `required` option in the test case ?